src/modules/battery.cpp

```diff
@@ -114,7 +114,7 @@ namespace modules {
 
     // Add formats and elements
     m_formatter->add(FORMAT_CHARGING, TAG_LABEL_CHARGING,
-        {TAG_BAR_CAPACITY, TAG_RAMP_CAPACITY, TAG_ANIMATION_CHARGING, TAG_LABEL_CHARGING});
+        {TAG_BAR_CAPACITY, TAG_RAMP_CHARGING, TAG_ANIMATION_CHARGING, TAG_LABEL_CHARGING});
```

Comment:
`TAG_RAMP_CAPACITY` should still be kept here. Otherwise all configs using it will break